js/main.js

```diff
@@ -309,3 +311,42 @@ $backButton.addEventListener('click', goBack);
 function goBack(event) {
   switchView('search-results');
 }
+
+function viewWatchlistCaptionUpdate() {
+  var $addToWatchlist = document.querySelector('.add-caption');
+  $addToWatchlist.textContent = 'View Watchlist';
+}
+
+function createEventListener() {
+  var $plusButtonBig = document.querySelector('.plus-button-big');
+  var $plusIconBig = document.querySelector('.plus-icon-big');
+  var $addToWatchlist = document.querySelector('.add-caption');
+  var $posterElement = document.querySelector('.poster-big');
+  var $titleElement = document.querySelector('.detailed-title');
+  var $yearElement = document.querySelector('.detailed-year');
+  var $genreElement = document.querySelector('.detailed-genre');
+  var $directorElement = document.querySelector('.detailed-director');
+  var $castElement = document.querySelector('.detailed-cast');
+  var $plotElement = document.querySelector('.detailed-plot');
+  var $idElement = document.querySelector('.detailed-id');
+  window.addEventListener('click', function (event) {
+    if ((event.target === $plusButtonBig) || (event.target === $plusIconBig) || (event.target === $addToWatchlist)) {
+      if ($addToWatchlist.textContent === 'Add to Watchlist') {
+        $plusIconBig.className = 'fas fa-check-circle plus-icon-big green';
+        $addToWatchlist.textContent = 'Added!';
+        setTimeout(viewWatchlistCaptionUpdate, 1000);
+        var cardDataForWatchlist = {};
+        cardDataForWatchlist.posterURL = $posterElement.getAttribute('src');
+        cardDataForWatchlist.title = $titleElement.textContent;
+        cardDataForWatchlist.year = $yearElement.textContent;
+        cardDataForWatchlist.genre = $genreElement.textContent;
+        cardDataForWatchlist.director = $directorElement.textContent;
+        cardDataForWatchlist.cast = $castElement.textContent;
+        cardDataForWatchlist.plot = $plotElement.textContent;
+        cardDataForWatchlist.id = $idElement.textContent;
+
+        data.savedCards.unshift(cardDataForWatchlist);
+      }
+    }
+  });
```

Comment:
You should apply this event listener in the global scope, it doesn't need to be done in this function